mcp-test/src/main/java/io/modelcontextprotocol/client/AbstractMcpAsyncClientTests.java

```diff
@@ -453,15 +453,15 @@ void testLoggingLevelsWithoutInitialization() {
 	@Test
 	void testLoggingLevels() {
 		withClient(createMcpTransport(), mcpAsyncClient -> {
-			Mono<Void> testAllLevels = mcpAsyncClient.initialize().then(Mono.defer(() -> {
-				Mono<Void> chain = Mono.empty();
+			Mono<Object> testAllLevels = mcpAsyncClient.initialize().then(Mono.defer(() -> {
+				Mono<Object> chain = Mono.empty();
 				for (McpSchema.LoggingLevel level : McpSchema.LoggingLevel.values()) {
 					chain = chain.then(mcpAsyncClient.setLoggingLevel(level));
 				}
 				return chain;
 			}));
 
-			StepVerifier.create(testAllLevels).verifyComplete();
+			StepVerifier.create(testAllLevels).expectNextCount(1).verifyComplete();
```

Comment:
What is the item? Does it have significance?

Reply:
it is null same as the ping

Reply:
This PR makes the setLevel call a `request` instead of `notification`. It should be a request according to the spec.
But a void request.